Gems/ROS2PoseControl/gem.json

```
@@ -1,6 +1,6 @@
 {
     "gem_name": "ROS2PoseControl",
-    "version": "3.0.0",
+    "version": "3.1.0",
```

Readme change should be treated as a _patch_. No new features in the codebase (_minor_), no breaking API (_major_).
```suggestion
    "version": "3.0.1",
```